LayoutTests/accessibility/mac/text-input-marked-range.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="../../resources/js-test-pre.js"></script>
+</head>
+<body>
+    <input id="editable" type="text"></input>
+    <script type="text/javascript">
+        description("This test ensures that the input method marked range is available to accessibility clients.");
+        editable.focus();
+        if (window.textInputController && window.accessibilityController) {
+            var element = accessibilityController.focusedElement;
+            shouldBe("element.textInputMarkedRange", "null");
+            textInputController.setMarkedText("test", 1, 0);
+            shouldBe("element.textInputMarkedRange", "'{0, 4}'");
+            textInputController.insertText("t");
+            shouldBe("element.textInputMarkedRange", "null");
+            textInputController.setMarkedText("test2", 1, 0);
+            shouldBe("element.textInputMarkedRange", "'{0, 5}'");
+        }
+    </script>
+    <script src="../../resources/js-test-post.js"></script>
+</body>
+</html>
```

Comment:
Optional nit: trailing newline?

Reply:
My editor usually does that for me, good catch!